.github/renovate.json

```
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "config:base",
+    ":reviewer(team:giantswarm/team-honeybadger)"
+  ],
+  "assigneesFromCodeOwners": false,
+  "dependencyDashboard": false,
+  "includePaths": [
+      ".github/workflows"
+  ],
+  "packageRules": [
+      "matchManagers": ["github-actions"]
+  ]
```

```suggestion
	"packageRules": [{
		"matchManagers": ["github-actions"]
	}]
```

JSON looks invalid here.

---

oh, obviously, my bad. It's fixed now.